deploy/buildtools/bitbuilder.py

```diff
@@ -282,8 +282,8 @@ def aws_create_afi(self) -> Optional[bool]:
                 break
             toplevel_superproject_path = next_higher_superproject
 
-        tags['firesim-origin'], tags['firesim-commit'], tags['firesim-ispushed'] = git_origin_sha_is_pushed(".")
-        tags['top-origin'], tags['top-commit'], tags['top-ispushed'] = git_origin_sha_is_pushed(toplevel_superproject_path)
+        tags['firesim-origin'], tags['firesim-commit'], tags['firesim-ispushed'] = git_origin_sha_is_pushed(".") #type: ignore
+        tags['top-origin'], tags['top-commit'], tags['top-ispushed'] = git_origin_sha_is_pushed(toplevel_superproject_path) #type: ignore
```

Comment:
If you have the changes in `util.git` do you still need the `#type: ignore` here?  I have a feeling that it was only needed because the typing was broken in git.py.

Reply:
It was complaining that the third return value is a bool:

```error: Incompatible types in assignment (expression has type "bool", target has type "str")```

I assume it's because the other values in the dictionary are strings. Changing the third return value from ```git_origin_sha_is_pushed()``` eliminated the error.

Reply:
I think it's better to annotate the dictionary to allow the values to be string or bool.  I did that in https://github.com/firesim/firesim/pull/1162/commits/ebee52eb99636d989f1e1c6d2970e7757e91104d